### PR TITLE
[doc] Add a description explaining that DELETE needs to be run in the batch mode

### PR DESCRIPTION
**Pull request by klion26**
### Purpose

Add a description to explaining that DELETE needs to be run in the batch mode

### Tests

no code change

### API and Format

no API/Format change

### Documentation

Change the documentation

### CODE REVIEW

**Comment by klion26** [comment]
Hi @JingsongLi , Please help to have a look when you're free, thanks.

Add the label above the code, because that it seems strange after adding label in the code block(such as below)

image

File: docs/content/concepts/file-operations.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ following sections:
 2. [File Layouts]({{< ref "concepts/file-layouts" >}}) and 
 3. How to use Paimon in [Flink]({{< ref "engines/flink" >}}).
 
+```
+Please execute the code in batch mode if there is a {{< label Batch >}} label above it, and you can add an entry `execution.runtime-mode: batch` in flink-conf.yaml to enable batch mode.
```
Comment (JingsongLi) [inline docs/content/concepts/file-operations.md:47]: I think we don't need this.

Comment (klion26) [reply docs/content/concepts/file-operations.md:47]: Ok, I can revert this.

**Comment by klion26** [comment]
@JingsongLi thanks for the review and merging!

**Pull request merged**